.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: Build test and deploy
+on:
+  pull_request:
+  workflow_dispatch:
```

Comment:
```suggestion
on:
  push:
    branches:
      - main
      - alpha
      - beta
  pull_request:
  workflow_dispatch:
```

Reply:
Sorry Giacomo, I was wrong when I said we didn't need the trigger on the branch. This way we enable CI only on branches where a release is scheduled.